sdk/appconfiguration/app-configuration/test/README.md

```diff
@@ -0,0 +1,41 @@
+# Testing
+
+To run our tests, first install the dependencies by running `npm install` (or `rush install`) at the root of the source code of this library.
+With the dependencies installed, you'll be able to run the unit tests and integration tests.
+
+## Unit tests
+
+To run unit tests, go to the root of this library and run `npm run unit-test`.
+This will run tests in a purely nodejs environment, and in a browser environment using [Karma](https://karma-runner.github.io/latest/index.html).
+These tests aim to reproduce the behavior of our library against remote endpoints through previously recorded HTTP request and responses.
+You won't need to have any environment configuration, nor internet access, for these tests to run.
+
+Keep in mind that if you modify the test files, the API shape of this library, or the underlying network calls, the recordings need to be re-generated.
+You can read more in the section: [Regenerating recordings](#regenerating-recordings).
+
+## Integration tests
+
+Our integration tests will run against the live resources, which are determined by the environment variables you provide.
```

Comment:
The unit-tests can run against the live environment, too, and the integration tests don't run with `TEST_MODE=live` by default, so I think this is a little misleading.

Reply:
I think we should address this in the test guidelines. These scripts should only do what they're supposed to do. I'll add a comment there.

Reply:
I've added feedback to myself on this PR: https://github.com/Azure/azure-sdk-for-js/pull/7784